kousa/lib/broth/message/auth/request.ex

```diff
@@ -134,6 +135,9 @@ defmodule Broth.Message.Auth.Request do
           :ok
       end
 
+      # subscribe to chats directed to oneself.
+      PubSub.subscribe("chat:" <> user_id)
```

Comment:
when you're authenticated, you subscribe to any chat messages that are directed straight to you.